src/Razor/src/Microsoft.AspNetCore.Razor.VSCode.Extension/package.json

```diff
@@ -175,6 +175,14 @@
           {
             "id": "razorDirectiveColon",
             "description": "A colon between directive attribute parameters"
+          },
+          {
+            "id": "razorDirective",
```

Comment:
Why only this inclusion of the semantic types? Or are the others already known to VSCode?